public/data/HlobisileLukhele.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Hlobisile Lukhele",
+    "location": "Durban, South Africa",
+    "bio": "I am Hlobisile, Fronted developer who is passionate about solving small businesses using my developing skills and passionate about building scalable websites",
+    "avatar": "https://github.com/<HlobisileLukhele>.png",
```

```suggestion
    "avatar": "https://github.com/HlobisileLukhele.png",
```
Do same with avatar.